src/Microsoft.Fx.Portability/ObjectModel/ApiInfoStorage.cs

```diff
@@ -13,6 +13,10 @@ public class ApiInfoStorage
         public string Type { get; set; }
         public string Name { get; set; }
```

Comment:
ApiInfoStorage is a type that our service uses to read the raw data from the catalog. In `CloudApiCatalogLookup`, we end up converting this to `ApiDefinition`.. So the class not used directly.